### PR TITLE
Add recipe for vapor

### PR DESCRIPTION
**Pull request by kciy**
Adds recipe for VAPOR package (https://github.com/connor-lab/vapor)

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/vapor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    - numpy >= 1.5.1
+
+test:
+  commands:
+    - vapor.py -h
+
+about:
+  home: "https://github.com/connor-lab/vapor"
+  license: GNU
+  license_family: GNU
```
Comment (bgruening) [inline recipes/vapor/meta.yaml]: Try GPL here

File: recipes/vapor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    - python >= 3
+    - numpy >= 1.5.1
+
+test:
+  commands:
+    - vapor.py -h
+
+about:
+  home: "https://github.com/connor-lab/vapor"
+  license: GNU
```
Comment (bgruening) [inline recipes/vapor/meta.yaml:32]: please add here GPL-3.0-only like https://spdx.org/licenses/GPL-3.0-only.html

I guess it its GPL 3 only according to https://github.com/connor-lab/vapor/blob/master/COPYING

File: recipes/vapor/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/vapor/meta.yaml]: you can also add a `license_file: COPYING` which references the file in the github repo.

File: recipes/vapor/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{% set name = "vapor" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "6d7818c6556d24663ffa72352a887f3e236f10bff4559bd86e4081c7cccbe8af" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/connor-lab/vapor/archive/refs/tags/VAPOR_1.0.2.tar.gz"
```
Comment (bgruening) [inline recipes/vapor/meta.yaml:10]: you can use here the template version number e.g.

`"https://github.com/connor-lab/vapor/archive/refs/tags/VAPOR_{{ version }}.tar.gz"`

File: recipes/vapor/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/vapor/meta.yaml]: ping

File: recipes/vapor/meta.yaml (hunk 2 above)
Comment (bgruening) [reply recipes/vapor/meta.yaml:32]: this is still valid

**Review by wm75 — APPROVED** [review]

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
@bioconda-bot please merge

**Pull request merged**